src/lib/sql/columns.sql

```diff
@@ -1,30 +1,22 @@
+-- We can uniquely point to a specific column using table_id + ordinal_position.
```

Comment:
This isn't included in this PR right?

Reply:
There's no explicit ID field for it. Do you want it to be separate field? (e.g. `${table_id}.${ordinal_position}`)

Reply:
Yeah I think something like that makes sense - saves the user from creating a composite key each time they want to update a column